### PR TITLE
feat: Print FUSEML_SERVER_URL instructions during installation

### PR DESCRIPTION
**Pull request by stefannica**

### CODE REVIEW

**Comment by jsuchome** [comment]
The value is already printed with current code, as part of _Deploying Core_ step...

**Comment by stefannica** [comment]
> The value is already printed with current code, as part of _Deploying Core_ step...

It is, but this PR makes it clear what that value should be used for. Here's an output example:

```
...
🚢 Deploying Core.........
✔️  FuseML core component deployed (http://fuseml-core.10.162.66.101.omg.howdoi.website).

🚢 Downloading command line client...
🚢 FuseML command line client saved as /home/user/fuseml/fuseml.
Copy it to the location within your PATH (e.g. /usr/local/bin).

🚢 
To use the FuseML CLI, you must point it to the FuseML server URL, e.g.:

    export FUSEML_SERVER_URL=http://fuseml-core.10.162.66.101.omg.howdoi.website

✔️  FuseML installed.
System domain: 10.162.66.101.omg.howdoi.website
```

File: paas/install_cli.go [hunk 1]
```diff
@@ -47,5 +48,11 @@ func downloadFuseMLCLI(ui *ui.UI) error {
 	ui.Note().Msg(fmt.Sprintf(
 		"FuseML command line client saved as %s.\nCopy it to the location within your PATH (e.g. /usr/local/bin).",
 		path))
+
+	fuseml_url := fmt.Sprintf("http://%s.%s", deployments.CoreDeploymentID, domain)
+	ui.Note().Msg(fmt.Sprintf(
+		"\nTo use the FuseML CLI, you must point it to the FuseML server URL, e.g.:\n\n    export FUSEML_SERVER_URL=%s",
```
Comment (flaviodsr) [inline paas/install_cli.go:54]: Just to keep it in line with the previous output, I would like to suggest removing the initial `\n`.

**Review by flaviodsr — CHANGES_REQUESTED** [review]

File: paas/install_cli.go (hunk 1 above)
Comment (stefannica) [reply paas/install_cli.go]: done

**Review by flaviodsr — APPROVED** [review]

**Comment by flaviodsr** [comment]
probably needs to be rebased

**Review by jsuchome — APPROVED** [review]

**Review by flaviodsr — APPROVED** [review]

**Pull request merged**